server/base/src/main/java/org/apache/accumulo/server/util/Admin.java

```diff
@@ -105,18 +105,12 @@ private static class SubCommandOpts {
   }
 
   @Parameters(
-      commandDescription = "signal the server process to shutdown normally, finishing anything it might be working on, but not starting any new tasks")
-  static class GracefulShutdownCommand extends SubCommandOpts {
-    @Parameter(required = true, names = {"-a", "--address"}, description = "<host:port>")
-    String address = null;
-  }
-
-  @Parameters(commandDescription = "stop the tablet server on the given hosts")
+      commandDescription = "Stop the servers at the given addresses allowing them to complete current task but not start new task.  When no port is specified uses ports from tserver.port.client property.")
```

Comment:
I think this needs to be updated to say that if `-f` is used, then the servers are stopped immediately and will not complete their current tasks.